blamo/lib/Export/index.dart

```diff
@@ -54,190 +73,402 @@ class _ExportPageState extends State<ExportPage> {
           ],
           backgroundColor: Colors.deepOrangeAccent
       ),
-      body: Center(
-        child: Center(child:_exportList(currentState,pickedDoc)),
-      ),
+
+      body: new Stack(
+        children: <Widget> [
+          new Container(
+            decoration: new BoxDecoration(
+              image: new DecorationImage(
+                  image: myImage.image,
+                  fit: BoxFit.cover,
+              )
+            )
+          ),
+          new Center(
+            child: Column(
```

Comment:
Exception thrown here when sending email for the first time, application continues to be usable save for I have only have been able to send one email over multiple app sessions so far. Could just be a local emulator/connection issue though, see if you can replicate?
![image](https://user-images.githubusercontent.com/43728787/77976634-5b50f980-72b2-11ea-8f4e-e44655f009f3.png)